### PR TITLE
Six named classes

### PR DESCRIPTION
**Pull request by quassy**

### CODE REVIEW

File: bin/01_train_classifier.py [hunk 1]
```diff
@@ -5,13 +5,16 @@
...
+
 
 # data generators for training and validation
 train_generator = classifier.get_image_datagen(folder=paths.TRAIN_DIR, batch_size=16)
 valid_generator = classifier.get_image_datagen(folder=paths.VALID_DIR, batch_size=16)
 
 # train model
-model = classifier.get_classifier(n_classes=5)
+classes = os.listdir(paths.TRAIN_DIR)
+model = classifier.get_classifier(n_classes=len(classes))
```
Comment (top-on) [inline bin/01_train_classifier.py:43]:
```suggestion
number_classes = len(os.listdir(paths.TRAIN_DIR))
model = classifier.get_classifier(n_classes=classes)
```
written this way, I would require less thinking :-)

File: bin/01_train_classifier.py [hunk 2]
```diff
@@ -5,13 +5,16 @@
 
 from traffic_cam import classifier, paths
 
+import os
```
Comment (top-on) [inline bin/01_train_classifier.py]: I suggest we import a build-in module earler.

**Review by top-on — APPROVED** [review]
feel free to merge after resolving the 2 comments

File: bin/01_train_classifier.py (hunk 2 above)
Comment (quassy) [reply bin/01_train_classifier.py]: PyCharm > Optimize Imports agrees with you ;)

File: bin/01_train_classifier.py (hunk 1 above)
Comment (quassy) [reply bin/01_train_classifier.py:43]: I thought we might need classes again, but... YAGNI!

**Pull request merged**